### PR TITLE
Monitoring e2e tests

### PR DESCRIPTION
**Pull request by kyoto**
e2e tests for 4.0 Monitoring UI.

### CODE REVIEW

File: frontend/integration-tests/protractor.conf.ts [hunk 1]
```diff
@@ -90,6 +90,7 @@ export const config: Config = {
     environment: ['tests/base.scenario.ts', 'tests/environment.scenario.ts'],
     secrets: ['tests/base.scenario.ts', 'tests/secrets.scenario.ts'],
     crud: ['tests/base.scenario.ts', 'tests/crud.scenario.ts', 'tests/secrets.scenario.ts', 'tests/filter.scenario.ts', 'tests/modal-annotations.scenario.ts', 'tests/environment.scenario.ts'],
+    monitoring: ['tests/base.scenario.ts', 'tests/monitoring.scenario.ts'],
```
Comment (alecmerdler) [inline frontend/integration-tests/protractor.conf.ts]: How are these expected to be ran as part of console CI? We should look at deploying `master` of the monitoring stack (Operator?) similar to what we do with OLM.

**Comment by kyoto** [comment]
@alecmerdler I added some more tests to this PR. I don't think we can run this as part of CI until CI is running 4.0, but I'd like to go ahead and get this merged so that the these tests can at least be run locally.

File: frontend/integration-tests/views/monitoring.view.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+// Details pages
+export const expiredSilenceIcon = $('.co-m-pane__details .fa-ban');
+export const ruleLink = $('.co-m-pane__details .co-resource-link__resource-name');
+export const silenceComment = $$('.co-m-pane__details dd').get(-2);
+export const firstAlertsListLink = $$('.co-resource-list__item a.co-resource-link').first();
+
+export const testDetailsPage = async(subTitle, alertName, expectLabel = true) => {
+  const heading = $('.co-m-nav-title .co-m-pane__name');
+  await wait(until.presenceOf(heading));
+  expect(heading.getText()).toContain(alertName);
```
Comment (alecmerdler) [inline frontend/integration-tests/views/monitoring.view.ts]: Do not have `expect()` calls in a view object function.

Comment (kyoto) [reply frontend/integration-tests/views/monitoring.view.ts]: I converted `testDetailsPage()` into a helper in `tests/monitoring.scenario.ts`.

**Comment by spadgett** [comment]
We should be able to enable these in CI when we switch to prow. AFAIK, monitoring will be there by default.

**Pull request merged**